systemvm/debian/opt/cloud/bin/cs/CsDhcp.py

```diff
@@ -84,7 +84,7 @@ def process(self):
     def configure_server(self):
         # self.conf.addeq("dhcp-hostsfile=%s" % DHCP_HOSTS)
         idx = 0
-        listen_address = ["127.0.0.1"]
+        listen_address = []
```

Comment:
if we have info of the type of network at this point we can make the initializer conditional. (just a thought)

Reply:
you want this change only apply on shared network ?

unfornately, the type of network is unknown at this point.